multiplex.go

```diff
@@ -150,7 +160,7 @@ func (mp *Multiplex) newStream(id streamID, name string) (s *Stream) {
 	s = &Stream{
 		id:          id,
 		name:        name,
-		dataIn:      make(chan []byte, 8),
+		dataIn:      make(chan []byte, 1),
```

Comment:
This will reduce the performance to a _crawl_.

Reply:
not really, it just puts backpressure earlier.

We could use MaxBuffers if you feel strongly about it.

Reply:
Yeah, maybe you're right. I saw "1" and thought we were being synchronous, but we're double-buffering (we have a _buffer_ of 1). Well, really, we're tripple-buffering:

1. The reader can be processing a buffer.
2. We can have a buffer waiting in data in.
3. We can be reading a buffer from the network.

So yeah, 1 is probably right.